sdk/core/azure-core/src/main/java/com/azure/core/implementation/jackson/ObjectMapperShim.java

```diff
@@ -42,15 +44,19 @@
     private static final int CACHE_SIZE_LIMIT = 10000;
 
     private static final Map<Type, JavaType> TYPE_TO_JAVA_TYPE_CACHE = new ConcurrentHashMap<>();
+    private static final Map<Type, MethodHandle> TYPE_TO_STRONGLY_TYPED_HEADERS_CONSTRUCTOR_CACHE
+        = new ConcurrentHashMap<>();
 
     /**
-     * Creates and configures JSON {@code ObjectMapper} capable of serializing azure.core types, with flattening and additional properties support.
+     * Creates and configures JSON {@code ObjectMapper} capable of serializing azure.core types, with flattening and
+     * additional properties support.
```

Comment:
nit: I suggest to adjust max line length to what checkstyle enforces and avoid re-formatting lines that were legit per checkstyle.

Reply:
So, this doesn't meet the line length limit but something is suppressing the check from failing.